test/NuGet.Core.Tests/NuGet.Build.Tasks.Pack.Test/PackTaskLogicTests.cs

```diff
@@ -581,6 +581,36 @@ public void PackTaskLogic_SupportsContentFiles_WithPackageFlatten()
             }
         }
 
+        [Fact]
+        public void PackTaskLogic_EmbedInteropAssembly()
```

Comment:
Can you add a DotnetIntegrationTest as well? 

The targets => PackRequest conversion is not covered and that one could easily be changed by someone in the future and the test wouldn't catch it :)